tests/client/net_handler_client.c

```diff
@@ -80,62 +82,38 @@ static int connect_tcp(robot_desc_field_t client) {
 
 /**
  * Function to receive data from net_handler on the UDP port
```

Comment:
Add a comment that it updates the global variable with the incoming DeviceData (Rn from the header it's hard to tell what the function output is)

Reply:
Fixed